### ISSUE TITLE
Document how to increase log levels

### ISSUE THREAD

**Issue by itowlson** [khenidak/kubernetes-dev-on-azure#1]
The tips/FAQ document advises: "make sure that you start your containers with the correct V parameters... **acs-engine default to use --v=2** change to higher value".

For beginners, it would be useful to describe how to specify a higher value when using acs-engine (or link to the docs if they exist elsewhere).

**Comment by khenidak** [comment]
I don't think you can do that via acs-engine api-model (AFAIK). You can change then build acs-engine which is probably more work than just change it on the cluster itself.

**Comment by itowlson** [comment]
Can you add documentation on how to change it on the cluster itself then, please, or link the FAQ advice to the documentation if you already have it?  It's not obvious to me, I'm afraid.

**Comment by khenidak** [comment]
Yes i will. for now, everything you need is /etc/kubernetes/manifests/kube-* on master. the container command should have one --v argument. 

Here is a sample for apiserver. Once you save the file kubelet will restart apiserver with the new spec.

```yaml

...
containers:
    - name: "kube-apiserver"
      image: "gcrio.azureedge.net/google_containers/hyperkube-amd64:v1.6.6"
      command:
        - "/hyperkube"
        - "apiserver"
        - "--admission-control=NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,ResourceQuota"
        - "--address=0.0.0.0"
        - "--allow-privileged"
        - "--insecure-port=8080"
        - "--secure-port=443"
        - "--cloud-provider=azure"
        - "--cloud-config=/etc/kubernetes/azure.json"
        - "--service-cluster-ip-range=10.0.0.0/16"
        - "--etcd-servers=http://127.0.0.1:2379"
        - "--etcd-quorum-read=true"
        - "--advertise-address=10.240.255.5"
        - "--tls-cert-file=/etc/kubernetes/certs/apiserver.crt"
        - "--tls-private-key-file=/etc/kubernetes/certs/apiserver.key"
        - "--client-ca-file=/etc/kubernetes/certs/ca.crt"
        - "--service-account-key-file=/etc/kubernetes/certs/apiserver.key"
        - "--storage-backend=etcd2"
        - "--v=4"
# here is  ^^^ my v param
....
```

**Comment by khenidak** [comment]
fixed by #5
